### ISSUE TITLE
Simple Desktop GUI (Initial Version)

### ISSUE THREAD

**Issue by bstoilov** [bstoilov/py3-pinterest#73]
Create a basic GUI that can be run under Windows, MacOs and Linux.

The following UI panels are requested by users of py3-pinterest
- login -> basic user information to be displayed and GUI way to add more pinterest accounts to the client
- Follow/Unfollow -> provides an easy way to take advantage of this functionality. It should be able to accept line separated list of users to follow/unfollow. Small delay between operations should be included so accounts don't get suspended.
- Board management -> panel that is able to list all boards and allow editing, deleting and creating.
- Pin management -> panel that can list all pins of a board and provide means to bulk edit and delete pins
- Search panel -> provides access to the search api of pinterest in the form of result list, from the list you should be able to create a list of pins to be repined (repin can be separate panel or maybe there is a better way)

**Comment by bstoilov** [comment]
@elmissouri16 hey this is the list of the highly requested features for the GUI. If you can create the basic project structure for the GUI. I will tag along and start helping as much as I can.
Probably it is best to create a new folder with the UI stuff inside?
A lot of people are also requesting scheduler. I can see this project as prerequisite for the schedulers as they will be easier to implement once we have some sort of UI.

**Comment by elmissouri16** [comment]
@bstoilov  yes new folder with all the UI stuff on it will be good I will start working on  it next few days ,

**Comment by imgVOID** [comment]
can u make a branch with the UI?

**Comment by bstoilov** [comment]
There already is but not much progress has been made
https://github.com/bstoilov/py3-pinterest/tree/gui-initial-version-setup
